### ISSUE TITLE
Get onto npm

### ISSUE THREAD

**Issue by chrisachard** [nanohop/sketch-to-react-native#11]
Right now installation requires git cloning this repo, but it should be hosted on npm also.

**Comment by flipace** [comment]
since you might plan on adding different target strategies (react, html..) it would probably be good to think of a more generic name before publishing. maybe something like **sketch2anything** ❓

**Comment by chrisachard** [comment]
Yes, good point.  I made the repo name before I really had the idea to target more inputs (.psd, .ai) and outputs (react, html, html-for-email, etc).  Unless I change the repo name then, I'm not sure what to do about that - I would want the npm name to match the repo name I think?

**Comment by flipace** [comment]
matching the repo name with the package name makes confusion less likely i guess. but a lot of large open source projects use monorepos nowadays where they put all their packages into one "main" repository, so there the names also don't quiet match...

not sure what's the best way to handle this right now, i guess the best thing would be to figure out a name before publishing (otherwise you'd have to add a deprecation warning and then point people to the new package)...

**Comment by chrisachard** [comment]
@flipace what about a name like "design-to-code"?  The name was available so I made a new repo: https://github.com/nanohop/design-to-code

I don't really want to just jump to a new repo though; maybe there should be this repo, and one for sketch-to-react, and then a more generic design-to-code?  I also don't want too much confusion, or things in too many places. Hm.

**Comment by Rob-Rychs** [comment]
Does the name really need to be specific to sketch? i liked the name in the package.json file svg-to-react, this also makes sense if you plan on adding react as an output and flags to the command to specify the output.... plus svg is the input file format it expects right?

**Comment by chrisachard** [comment]
Sketch SVGs are slightly different than .ai svgs, etc - so right now it only takes SVGs generated from Sketch.  I may have limited the library too much with the choice of name however.  

It turns out that renaming a library is really easy (I didn't know that until just now)... so it would be worth figuring out if there was a better name that opened it up for the future.  

- sketch-to-react would be a good name for both react and react-native in my opinion. 
- design-to-react might be a good generic term.  
- I already referenced design-to-code which is about the most generic repo name I could come up with.

I'll say that I am a bit hesitant to rename it to something too generic, because I think there is value in a name that's very specific (so people know exactly what it does).  Does anyone have ideas about renaming repos (pros and cons), and what a new name could be?

**Comment by geminiyellow** [comment]
name is not important, 
everybody just want to use this tool but not clone repo.
you can naming it to `the-tool-make-you-very-happy-but-its-name-is-very-looooooooooog-but-i-love-it`. please just publish it.
